internal/gensupport/media_test.go

```diff
@@ -149,27 +149,29 @@ func TestDetermineContentType(t *testing.T) {
 
 func TestNewInfoFromMedia(t *testing.T) {
 	const textType = "text/plain; charset=utf-8"
+	chunkBuffer := make([]byte, 0)
```

Comment:
Could you add a test that will make use of the `MediaBuffer`? Maybe you could do something with `ResumableUpload`?

Reply:
Sure, test case added